openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-packetviewer/src/tools/PacketViewer/PacketViewer.vue

```diff
@@ -491,31 +491,49 @@ export default {
       }
     },
     packetChanged(event) {
-      this.api.get_target(event.targetName).then((target) => {
-        this.ignoredItems = target.ignored_items
-      })
       this.api
-        .get_packet_derived_items(event.targetName, event.packetName)
-        .then((derived) => {
-          this.derivedItems = derived
+        .get_target(event.targetName)
+        .then((target) => {
+          if (target) {
+            this.ignoredItems = target.ignored_items
+
+            return this.api.get_packet_derived_items(
+              event.targetName,
+              event.packetName,
+            )
+          } else {
+            // Probably got here from an old config or URL params that point to something that no longer exists
+            // (e.g. the plugin that defined this target was deleted). Unset these to avoid API errors.
+            this.targetName = null
+            this.packetName = null
+            this.$router.push({
+              name: 'PackerViewer',
+              params: {},
+            })
```

Comment:
Nice ... I think there's probably other places in COSMOS where we're "remembering" previous values but the most obvious is here in Packet Viewer